app/assets/src/components/views/Landing.jsx

```diff
@@ -8,6 +8,7 @@ import {
   Grid,
   Image,
   Message,
+  Container,
```

Comment:
I actually missed this. This goes agains our policy of wrapping semantic ui element in our elements.